src/config.rs

```diff
@@ -1258,6 +1258,64 @@ impl Package {
         })
     }
 
+    /// will map aliased imports to full path in the actual body of the document
+    /// and return the new document body as string
+    ///
+    /// [-- import: some_alias] will be mapped to
+    /// [-- import full_path_of_some_alias as some_alias]
+
+    pub fn fix_imports_in_body(&self, body: &str, id: &str) -> ftd::p1::Result<String> {
+        let mut new_body = String::new();
+
+        let mut ln = 1;
+        for line in body.lines() {
+            let line_string = line.trim();
+
+            // Import statement (All possible cases)
+            // [-- import: full_path] (pass this)
+            // [-- import: alias] -> convert this to -> [--import: full_path as alias]
+            // [-- import: full_path as alias] (pass this)
+            let final_line =
+                if line_string.starts_with("-- import") && !line_string.contains(" as ") {
```

Comment:
You have to consider import with `as` also. Let's consider

```ftd
; in FPM.ftd
-- fpm.dependency: fifthtry.github.io/doc-site as ds

; in some file

-- import: ds/page as p
```

You have to make this case works too.


```